app/src/main/java/com/babylon/wallet/android/presentation/accessfactorsources/deriveaccounts/DeriveAccountsViewModel.kt

```diff
@@ -180,7 +186,7 @@ class DeriveAccountsViewModel @Inject constructor(
             accessFactorSourcesIOHandler.setOutput(
                 output = AccessFactorSourcesOutput.DerivedAccountsWithNextDerivationPath(
                     derivedAccounts = derivedAccounts,
-                    nextDerivationPathOffset = indicesToScan.last() + 1u
+                    nextDerivationPathIndex = HdPathComponent.init(indicesToScan.last().indexInGlobalKeySpace + 1u)
```

Comment:
there is no label in `HdPathComponent.init` hmm so it is not clear that that init uses `globalKeyspace`.  can you spell out `globalKeySpace = ` ?

Reply:
Sure